### PR TITLE
28 add obs explainer page

### LINKED ISSUE #28: Add Obs Explainer page from cop26 for needs text update

### LINKED ISSUE #36: `extraStyling` warnings in dev tools
We're getting an error in devtools:

`Warning: React does not recognize the `extraStyling` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `extrastyling` instead.`

We're passing in this styling to the `TextButton` component. Maybe we can avoid this by using separate CSS classes.

### LINKED ISSUE #37: Unable to select different emissions in `EmissionsBox`
I can't select different emission types on the explainer page, I get the following error:

```
$(name) not in images 2 EmissionsBox.js:58
    setImage EmissionsBox.js:58
    React 14
    unstable_runWithPriority scheduler.development.js:468
    React 4
```

### PR DESCRIPTION
**Pull request by SutarPrasad**
* **Describe the changes implemented in this PR** (Bug fix, feature, docs update, ...)
  Added Obs Explainer page and needs updating the text along with the FAQ page.

* **Please check if the PR fulfils these requirements**

- [x] Closes #28 
- [x] Closes #37 
- [x] Closes #36
- [ ] ESLINT passed

### CODE REVIEW

**Review by gareth-j — CHANGES_REQUESTED** [review]
Just a few changes and questions.

File: src/Dashboard.js [hunk 1]
```diff
@@ -338,17 +339,29 @@ class Dashboard extends React.Component {
                 toggleOverlay={this.toggleOverlay}
                 closePanel={this.toggleSidebar}
               >
+                <Button variant="contained" >
```
[the remark this answers is not in the document]
Comment (SutarPrasad) [reply src/Dashboard.js:342]: image

I am in process to update to above view for those

File: src/components/FAQ/FAQ.js [hunk 2]
```diff
@@ -1,7 +1,8 @@
 import React from "react";
 
 import QnA from "../QnA/QnA";
-import styles from "./FAQ.module.css";
+import styles from "../../Dashboard.module.css";
```
[the remark this answers is not in the document]
Comment (SutarPrasad) [reply src/components/FAQ/FAQ.js:4]: Yeah, Need to check this, Will get this reverted

File: src/Dashboard.js [hunk 3]
```diff
@@ -338,17 +338,16 @@ class Dashboard extends React.Component {
                 toggleOverlay={this.toggleOverlay}
                 closePanel={this.toggleSidebar}
               >
-                <Link to="/" className={styles.navLink}>
```
Comment (gareth-j) [inline src/Dashboard.js:341]: Keep these links for now but add in the link to the Explainer page.

**Review by gareth-j — CHANGES_REQUESTED** [review]
So some changes needed to this one. We should move the changes/restyling of the sidebar out to a separate PR as it doesn't under adding the explainer page. Then just a bit of padding needeed on the FAQ page and the extra route removing.

File: .github/workflows/node.js.yml [hunk 4]
```diff
@@ -21,6 +21,7 @@ jobs:
       uses: actions/setup-node@v1
       with:
         node-version: ${{ matrix.node-version }}
+    - run: npm cache clean --force
```
[the remark this answers is not in the document]
Comment (SutarPrasad) [reply .github/workflows/node.js.yml]: cleans pre-cached data, incase if there is any old unwanted package that has gotten cached. Sometimes the heap memory can arise from this too.

File: src/Dashboard.js [hunk 5]
```diff
@@ -338,17 +338,16 @@ class Dashboard extends React.Component {
...
-                <Link to="/FAQ" className={styles.navLink}>
-                  FAQ
-                </Link>
+                <MyTabs/> 
+
+
               </ControlPanel>
             </aside>
             <Routes>
+              <Route path="/explainer" element={<Explainer></Explainer>}/>
```
[the remark this answers is not in the document]
Comment (SutarPrasad) [reply src/Dashboard.js:347]: Might have been created inadvertently

File: src/components/FAQ/FAQ.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (SutarPrasad) [reply src/components/FAQ/FAQ.js:4]: done

File: src/components/MyTabs/MyTabs.js [hunk 6]
```diff
@@ -0,0 +1,50 @@
+import React, { useState } from "react";
```
[the remark this answers is not in the document]
Comment (SutarPrasad) [reply src/components/MyTabs/MyTabs.js]: done created new PR just for the styling

File: src/Dashboard.js (hunk 3 above)
Comment (SutarPrasad) [reply src/Dashboard.js]: done

File: src/components/Explainer/Explainer.js [hunk 7]
```diff
@@ -1,106 +1,106 @@
...
-//     const body = `To start with we can use our inventory emissions as our initial “best guess”
-//     and compare this to real atmospheric measurements. We can then run simulations to make 
-//     small changes to the possible emissions and continuously improve to better match the measurements made at each site.
-//     The animation below illustrates this process, using a site in Tacolneston, Norfolk which is part of
-//     the UK DECC network as an example.`;
-//     return <ExplanationBox header={header} intro={body} />;
-//   }
+  createComparisonExplainer() {
+    const header = "Comparing models with observations";
+    const body = `To start with we can use our inventory emissions as our initial “best guess”
```
Comment (rt17603) [inline src/components/Explainer/Explainer.js:55]:
```suggestion
    const body = `To start with we can use inventory emissions as our initial “best guess”
```

File: src/components/Explainer/Explainer.js [hunk 8]
```diff
@@ -1,106 +1,106 @@
...
+    and compare this to real atmospheric measurements. We can then run simulations to make 
+    small changes to the possible emissions and continuously improve to better match the measurements made at each site.
```
Comment (rt17603) [inline src/components/Explainer/Explainer.js:57]:
```suggestion
    repeated small changes to the possible emissions to better match the measurements made at each site.
```

File: src/components/Explainer/Explainer.js [hunk 9]
```diff
@@ -1,106 +1,106 @@
...
+    The animation below illustrates this process, using a site in Tacolneston, Norfolk which is part of
```
Comment (rt17603) [inline src/components/Explainer/Explainer.js:58]:
```suggestion
    The animation below illustrates this process, using the UK DECC network site in Tacolneston, Norfolk.
```

File: src/components/Explainer/Explainer.js [hunk 10]
```diff
@@ -1,106 +1,106 @@
...
+    the UK DECC network as an example.`;
```
Comment (rt17603) [inline src/components/Explainer/Explainer.js:59]:
```suggestion
```

**Comment by SutarPrasad** [comment]
> So some changes needed to this one. We should move the changes/restyling of the sidebar out to a separate PR as it doesn't under adding the explainer page. Then just a bit of padding needeed on the FAQ page and the extra route removing.

All of these changes have been addressed and changes are implemented as advised

File: src/components/EmissionsBox/EmissionsBox.js [hunk 11]
```diff
@@ -1,197 +1,200 @@
...
+      },
+    };
+
+    this.setImage = this.setImage.bind(this);
+    this.setSpecies = this.setSpecies.bind(this);
+    this.state = { images: images, selectedSector: "Total", selectedSpecies: "CO2" };
+  }
+
+  setImage(e) {
+    const name = e.target.dataset.onclickparam;
```
[the remark this answers is not in the document]
Comment (SutarPrasad) [reply src/components/EmissionsBox/EmissionsBox.js:56]: agreed, will make not of this as a new issue.

**Pull request merged**